packages/dnb-eufemia/src/extensions/forms/Field/OrganizationNumber/OrganizationNumber.tsx

```diff
@@ -82,6 +82,9 @@ function OrganizationNumber(props: Props) {
  * www.brreg.no/om-oss/registrene-vare/om-enhetsregisteret/organisasjonsnummeret/
  */
 function isValidOrgNumber(digits: string) {
+  if (digits === '000000000') {
```

Comment:
```suggestion
  if (parseFloat(digits) === 0) {
```

To support other countries, I think we simply can parse it.